frameHeader.go

```diff
@@ -176,7 +176,9 @@ func (frh *FrameHeader) readFrom(br *bufio.Reader, max uint32) (int64, error) {
 
 	// Parsing FrameHeader's Header field.
 	frh.parseValues(header)
-
+	if int(frh.kind) > len(framePools) {
```

Comment:
First, I prefer to use a fixed number, let's say `MaxFrameType = 0x10` and even with that, the comparison should be `>=`, right?

Reply:
And WDYT to use the `FrameContinuation` constant as we already use this one to create the framePools array?

Reply:
Yes, my bad, can you add the MaxFrameType, please? That way the code is a little bit clearer.

Reply:
I retract that. The FrameContinuation is fine.